setup.py

```diff
@@ -32,7 +53,7 @@
     ],
     install_requires=[
         'panoramic-auth>=0.2.0',
-        'requests>=2.24.0',
+        'requests<2.24.0',
```

Comment:
Do we need a strict version here? Shouldn't it be `<=`?

Reply:
I could have put a comment here, but the problem is 
`error: requests 2.24.0 is installed but requests<2.24.0 is required by {'snowflake-connector-python'}`

Reply:
btw added comment there